### PR TITLE
Fix around session detail behavior.

### PR DESCRIPTION
**Pull request by Kurogoma4D**
## Description

- Fix speaker icon.
- Hide X link button when link is empty.
- Use root Navigator to show session detail page.

Fixes #issue_number

## Type of change

Please delete options that are not relevant.

- [ ] New feature
- [x] Bug fix
- [ ] Refactor
- [ ] Style
- [ ] Documentation
- [ ] CI/CD
- [ ] Other

## How Has This Been Tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.

## Checklist

- [x] My code follows the style guidelines of this project.
- [x] My code has been formatted with `flutter format lib`.
- [x] My code has been fixed with `dart fix --apply lib`.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Kurogoma4D** [comment]
セッション詳細のナビゲーション方法を変更した経緯：
- `Router.go` を使った遷移の場合、`Router.pop` を使って戻ろうとしてもクエリパラメータの情報が保持されない
- iOSで戻るジェスチャーが効かない（ドロワーの判定に吸われてしまう）

File: lib/main.dart [hunk 1]
```diff
@@ -11,12 +11,15 @@ import 'package:firebase_crashlytics/firebase_crashlytics.dart';
...
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
   updatePathStrategy();
+  GoRouter.optionURLReflectsImperativeAPIs = true;
```
Comment (koji-1009) [inline lib/main.dart:21]: 👀 

https://pub.dev/documentation/go_router/latest/go_router/GoRouter/optionURLReflectsImperativeAPIs.html

---

session詳細画面にpushで遷移しているため、画面遷移時のURL変更をURLバーに反映するってことですね。

File: lib/ui/screen/sessions/sessions.dart [hunk 2]
```diff
@@ -197,7 +197,7 @@ class _SessionCard extends ConsumerWidget {
           SessionTalk() || SessionSponsor() => () {
               SessionDetailRoute(
                 sessionId: session.id,
-              ).go(context);
+              ).push(context);
```
Comment (koji-1009) [inline lib/ui/screen/sessions/sessions.dart:200]: 👍

**Review by koji-1009 — APPROVED** [review]
LGTM

**Pull request merged**